baker/postUpdatedHook.ts

```diff
@@ -17,12 +17,24 @@ const zeroDateString = "0000-00-00 00:00:00"
 const syncPostToGrapher = async (
     postId: number
 ): Promise<string | undefined> => {
-    const rows = await wpdb
-        .getSingleton()
-        .query(
-            "SELECT * FROM wp_posts WHERE ID = ? AND post_status != 'trash'",
-            [postId]
-        )
+    const rows = await wpdb.getSingleton().query(
+        `--sql
+        -- we select all the fields from wp_posts and then we also
+        -- json array aggregate the authors. The authors come as strings like
+        -- Charlie Giattino Charlie Giattino Charlie Giattino 44 charlie@ourworldindata.org
```

Comment:
how did that happen??

Reply:
It's a plugin that does this. Don't ask :)